app/assets/javascripts/playspaces.js

```diff
@@ -7,27 +7,35 @@ var PlayspaceSidebar = function () {
 $(document).ready(function(){
   var option = {
     HK: ["Central", "North Point", "Repulse Bay", "The Peak", "Wan Chai", "Western"],
-    KLN: ["Ho Man Tin", "Tsim Sha Tsui", "Wong Tai Sin"],
+    Kln: ["Ho Man Tin", "Tsim Sha Tsui", "Wong Tai Sin"],
     NT: ["Sai Kung", "Sha Tin", "Tai Po"]
   }
 
-  $("#playspace_region").on('change', function() {
-    $("#playspace_district").html("");
-    if ($("#playspace_region").val() == "HK") {
-      for (var i = 0; i < option["HK"].length; i++) {
-        $("#playspace_district").append("<option value='"+option["HK"][i]+"'>"+option["HK"][i]+"</option>")
+  var bindRegionDistrict = function (formID) {
+    $(formID + " #playspace_region").on('change', function() {
+      $(formID + " #playspace_district").html("");
+      $(formID + " #playspace_district").append("<option value>Select by district:</option>")
+
+      if ($(formID + " #playspace_region").val() == "HK") {
+        for (var i = 0; i < option["HK"].length; i++) {
```

Comment:
var bindRegionDistrict = function (formID) {

  $(formID + " #playspace_region").on('change', function() {
    $(formID + " #playspace_district").html("");
    $(formID + " #playspace_district").append("<option value>Select by district:</option>")


    option.each |r|
      for (var i = 0; i < option[r].length; i++) {
        $(formID + " #playspace_district").append("<option value='"+option[r][i]+"'>"+option[r][i]+"</option>")
      }
    }

  })
};

  bindRegionDistrict('#search')
  bindRegionDistrict('#new_playspace')

})